### ISSUE TITLE
Create Specification Documents

### ISSUE THREAD

**Issue by endymuhardin** [BelajarJavaBareng/bpustaka#36]
Create specification documents displaying final UI mockup.
Document outline : 

# Overview

Explains the purpose of the application, who will use it, what task can be accomplished by using the apps.

# Functional Requirement

This section explains about functional requirement

## Feature 1

* Put UI Mockup here
* Explain who will use this screen (user level / permission)
* Explain what is the outcome of using this screen

# Non Functional Requirement

List other requirement here if any, such as : 
* performance
* configurability
* security
* ease of use
* etc

**Comment by justhardinal** [comment]
Feature 1
#Login
![login at home screen](https://cloud.githubusercontent.com/assets/11325703/6724305/bb97a986-ce27-11e4-8d46-79ff8d11124a.png)

this screen was using for all user( member, librarian) and outcome of this screen is dashboard of member for borrowing and return the book, or librarian for manage member data, transaction, report.etc.
#Regitstration 
![new registration form](https://cloud.githubusercontent.com/assets/11325703/6724348/d603855a-ce28-11e4-8361-acff5d39c7b3.PNG)
this screen was using for register all of prospective members to join to be a member and get facilitate of borrowing and return the books, the output was member data

**Comment by dhaowoods** [comment]
beberapa sudah sekalian saya pindah ke readme.md'nya mas @justhardinal >.<

**Comment by justhardinal** [comment]
yup @dhaowoods i knew it, but of the all screen .png, there are some screen was the same function but different design ..so i think that we choose the good design and explain in this issue :+1:  :) to fix and finished this UI-MockUp

**Comment by justhardinal** [comment]
search book for any user
![book search list version](https://cloud.githubusercontent.com/assets/11325703/6893068/f8aa487c-d6f3-11e4-9ffb-092c934fe12c.PNG)
![book search thumbnail version](https://cloud.githubusercontent.com/assets/11325703/6893069/f8aca202-d6f3-11e4-960e-aa03353e9724.PNG)

**Comment by Asrul09** [comment]
i think the first is better mas @justhardinal

**Comment by justhardinal** [comment]
how about this borrowing transaction form
![peminjaman](https://cloud.githubusercontent.com/assets/11325703/6958213/0d3685a8-d933-11e4-923c-ea3284d161bf.PNG)

**Comment by justhardinal** [comment]
schema of database org.bpustaka
![skemadbbpustaka](https://cloud.githubusercontent.com/assets/11325703/7108680/07ba2156-e1b8-11e4-8cb6-a2b9af58922a.png)

if there is any correction please  put in comment, thx :)
